File: simple-dpgf-api/src/main/java/com/simpledpgfapi/user/model/user/dto/UserDetailsDto.java
package com.simpledpgfapi.user.model.user.dto;

import com.simpledpgfapi.user.model.role.RoleEnum;
import lombok.Data;

@Data
public class UserDetailsDto {
    private String id;
    private String firstName;
    private String lastName;
    private String email;
    private RoleEnum role;
}

File: simple-dpgf-webfront/src/app/core/dtos/user/UserDetailsDto.ts
//import { InvitationStatusEnum } from "../../enums/InvitationStatusEnum";

import { RoleEnum } from "../../enums/RoleEnum";

export interface UserDetailsDto {
  id: string;
  firstName: string;
  lastName: string;
  email: string;
  role: RoleEnum;
}
